.github/workflows/test-application.yaml

```diff
@@ -0,0 +1,86 @@
+name: Test application
+
+on:
+    pull_request:
+    push:
+        branches:
+            - 'master'
+            - '[0-9]+.x'
+            - '[0-9]+.[0-9]+'
+            - '[0-9]+.[0-9]+.x'
```

Comment:
i'd say remove these, as we won't be supporting separate branches.

Reply:
done. and tests are all green now: https://github.com/alexander-schranz/phpcr-shell/pull/1/checks